inst/EpiModel2.rmd

```diff
@@ -482,13 +543,13 @@ param <- param.net(inf.prob = 0.4, act.rate = 1, rec.rate = 0.02,
 The input parameters for departures do not need to be updated however users should be aware of the change in interpretation. As with the previous SI example, control settings for `depend` and `save.network` (and `save.transmat`) should be updated to `resimulate.network` and `tergmLite` respectively.
 
 ```{r echo = TRUE, eval = FALSE}
-## EpiModel 1.x Workflow
+## EpiModel 1.x
 control <- control.net(type = "SIR", nsteps = 100, depend = FALSE, 
                        save.network = FALSE)
 ```
 
 ```{r echo = TRUE, eval = FALSE}
-## EpiModel 2.0 Workflow
+## EpiModel 2.0
 control <- control.net(type = "SIR", nsteps = 100, resimulate.network = FALSE,
                        tergmLite = TRUE)
 ```
```

Comment:
suggestions for lines 564-569: copy-edits

for all of the code below, change to conventional `# EpiModel 1.x` and `# EpiModel 2.0` and use ## for the comments after

> ## User-Defined Modules: SEIR Epidemic Model
> Our final example is an extension of the SIR model not represented in base EpiModel: an SEIR epidemic that introduces an exposed state to the epidemic process, necessitating an updated infection module and a new disease progression module. Below, we have the new infection module that is based on EpiModel 1.x syntax.
> ```{r echo = TRUE, eval = FALSE}
> # EpiModel 1.x

suggestions for line 794: 
In EpiModel 2.0, if any base modules have been changed or any user-defined modules are present the user must specify their epidemic type as `NULL`, even if the module potentially follows an acceptable base epidemic type (SI, SIR, or SIS). Further, the user must specify all base epidemic modules, even if these modules are not used within the epidemic process. This change has been made in order to make clear the difference between base EpiModel use (in which most of the modeling setup is automated) and more advanced EpiModel use (including module extension). The above code written in EpiModel 1.x should then be updated to: